scripts/checkParameterCase.py

```diff
@@ -10,66 +9,65 @@ def parse_schema_element(root,
                          path,
                          xsd='{http://www.w3.org/2001/XMLSchema}',
                          recursive_types=['PeriodicEvent', 'SoloEvent', 'HaltEvent']):
-  """
+    """
```

Comment:
Looks like docstrings are not properly aligned after changing indentation: https://github.com/google/yapf/issues/575

We should probably do it manually once

Reply:
Agreed, and it seems that it avoids them intentionally.  I'll do a pass to fix these